documents/moving-across-committees.md

```diff
@@ -0,0 +1,25 @@
+# Transfering a Working Group from the Node.js TSC to the Node.js Community Committee
+
+As interested, Working Groups that are focused on serving, growing, and building the community surrounding Node.js may decide that it would like to become a part of the Node.js Community Committee. Similarly, if a Working Group under the Node.js Community Committee is focused on more technical tasks and may want to transfer over to the TSC.
+
+This document is a guide on how to begin the transfer process from one committee to another.
+
+## Moving from the TSC to the Community Committee
+* **Step 1**: Create an issue in [Community Committee](https://github.com/nodejs/community-committee/issues) repository asking if the Community Committee would be willing to take the Working Group in question on as a Community Committee Working Group.
+  * **Step 1.1**: If there are no objections, continue on to step two.
+* **Step 2**: Create an issue in the repository for the Working Group in question, proposing the move change. Given the governance model, this will likely be a vote and will need majority approval of the Working Group's members.
+* **Step 3**: Once approved, you will need to create issues to be removed from the TSC and added to the Community Committee, and an issue tracking the change in the relevant WG repo. 
```

Comment:
replace `issue` with `pull request`, and there's no need for a separate issue in the WG repo. We should use the original issue creating in what I suggest be step 1. Otherwise, we're creating notification clutter.